airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/class_types_registry.py

```diff
@@ -17,5 +26,12 @@
     "MinMaxDatetime": MinMaxDatetime,
     "NextPageUrlPaginator": NextPageUrlPaginator,
     "OffsetPaginator": OffsetPaginator,
+    "NoPaginator": NoPagination,
```

Comment:
it's a Null object to avoid the need for multiple None checks. Essentially a typed and non-generic version of https://www.oreilly.com/library/view/python-cookbook/0596001673/ch05s24.html

Reply:
having the object makes sense. Does the developer need to pass it in their YAML?

Reply:
got it. removed from the map because it's probably not needed